websites/A/Averge/metadata.json

```diff
@@ -0,0 +1,22 @@
+{
+    "$schema": "https://schemas.premid.app/metadata/1.3",
+    "service": "Averge.EU",
+    "author": {
+      "name": "lordreider",
+      "id": "853251440763600897"
+    },
+    "category": "other",
+    "logo": "https://averge.eu/cdn/img/averge.png",
+    "thumbnail": "https://averge.eu/cdn/img/averge.png",
+    "version": "1.0.0",
+    "color": "#0000ff",
+    "tags": [
+      "security",
+      "law"
+    ],
+    "url": "averge.eu",
+
+    "description": {
+      "en": "Averge n.e.V. - Verein zur Bekämpfung rechtswidriger Inhalte im Internet."
```

Comment:
This is not an English description. Use the correct language code and provide an English translation